variables.tf

```diff
@@ -12,57 +15,28 @@ variable "business-unit" {
   default     = "mojdigital"
 }
 
-variable "namespace" {
+variable "is-production" {
+  default = "false"
 }
 
-variable "engine_type" {
-  description = "Engine used e.g. postgres"
-  default     = "postgres"
+variable "infrastructure-support" {
+  description = "The team responsible for managing the infrastructure. Should be of the form <team-name> (<team-email>)"
 }
 
-variable "engine_version" {
-  description = "The engine version to use e.g. 9.6"
-  default     = "9.6"
+variable "namespace" {
 }
 
 variable "instance_type" {
   description = "replication instance size, e.g dms.t2.medium"
   default     = "dms.t2.medium"
 }
 
+variable "allocated_storage" {
+  description = "how many GB for local buffer"
+  default     = 32
+}
+
 variable "aws_region" {
   description = "Region into which the resource will be created."
   default     = "eu-west-2"
 }
-
-variable "source_database_name" {
-  description = "Name of source database (psql -d)"
-}
-
-variable "source_database_username" {
-  description = "username in source database (psql -U)"
-}
-
-variable "source_database_password" {
-  description = "user's password in source database"
-}
-
-variable "source_database_host" {
```

Comment:
I would actually prefer these variables to exist. They save the need for "magic strings" and you can add a description to their meaning.